src/components/posts/NewPostButton.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <v-dialog 
+    v-model="dialog"
+    max-width="800"
+    persistent
+  >
+    <template v-slot:activator="{on, attrs}">
+      <v-fab-transition>
+        <v-btn
+          v-bind="attrs"
+          class="new_post_button"
+          color="primary"
+          dark
+          block
+          v-on="on"
+        >
+          <v-icon>mdi-pencil</v-icon>
+          Add new post
```

Comment:
use i18n
```suggestion
        {{ $t('posts..create.add-new-post') }}
```

translation key is only a suggestion